### PR TITLE
Non native build

### PR DESCRIPTION
**Pull request by darkfiberiru**
Non native build is done  with platform and arch flags in manifest

{
  "arch": {
    "platform": "arm64",
    "arch": "aarch64"
  },

### CODE REVIEW

**Comment by darkfiberiru** [comment]
Doesn't handle iso/image creation for non-amd64 build yet

File: scripts/build.sh [hunk 1]
```diff
@@ -29,13 +29,6 @@
 
 export PATH="/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
 
-delete_tmp_manifest(){
```
Comment (beanpole135) [inline scripts/build.sh:32]: I think you need to check your merge history - this section should not be getting removed.

Comment (darkfiberiru) [reply scripts/build.sh:32]: Returned

**Comment by darkfiberiru** [comment]
@kmoore134 @beanpole135 Please let me know what you think now and if I need any more cleanup.

Also let me know if you want me to redo this entire commit history again with out removing and re-adding the delete_tmp_manifest. Functional no different just dirties commit history a little

**Review by kmoore134 — APPROVED** [review]
Looks much better!

**Pull request merged**